okhttp-urlconnection/src/main/java/okhttp3/internal/huc/OkHttpURLConnection.java

```diff
@@ -54,7 +54,6 @@
 import okhttp3.internal.Version;
 import okhttp3.internal.http.HttpDate;
 import okhttp3.internal.http.HttpHeaders;
-import okhttp3.internal.http.HttpMethod;
```

Comment:
Do you mind reverting the changes to OkHttpUrlConnection? This is likely to be backwards-incompatible